requirements/dev.in

```diff
@@ -0,0 +1,68 @@
+-r prod.in
+
+async_generator==1.10
+black==21.12b0
+blessings==1.7
+bpython==0.22.1
+braceexpand==0.1.7
+chkcrontab==1.7
+click==8.0.3
+cookies==2.2.1
+coverage==6.2
+cssselect==1.1.0
+curtsies==0.3.10
+cwcwidth==0.1.6
+diff-match-patch==20200713
+entrypoints==0.3
+factory-boy==3.2.1
+Faker==11.3.0
+flake8==4.0.1
+h11==0.13.0
+iniconfig==1.1.1
+isort==5.10.1
+mccabe==0.6.1
+more-itertools==8.12.0
+mypy-extensions==0.4.3
+outcome==1.1.0
+pathspec==0.9.0
+pep8==1.7.1
+pipdeptree==2.2.1
+platformdirs==2.4.1
+pluggy==1.0.0
+py==1.11.0
+pycodestyle==2.8.0
+pyflakes==2.4.0
+Pygments==2.11.2
+PyPOM==1.0
+pyquery==1.4.3
+pytest-base-url==1.4.2
+pytest-cov==3.0.0
+pytest-django==4.5.2
+pytest-html==3.1.1
+pytest-metadata==1.11.0
+pytest-mock==3.6.1
+pytest-parallel==0.1.1
+pytest-rerunfailures==10.2
+pytest-selenium==1.17.0
+pytest-variables==1.9.0
+pytest==6.2.5
+pyxdg==0.27
+regex==2022.1.18
+responses==0.17.0
+selenium==3.141.0
+sniffio==1.2.0
+sortedcontainers==2.4.0
+tblib==1.7.0
+tenacity==6.3.1
+text-unidecode==1.3
+toml==0.10.2
+tomli==1.2.3
+translate-toolkit==3.5.3
+trio-websocket==0.9.2
+trio==0.19.0
+wcwidth==0.2.5
+wsproto==1.0.0
+
+# Related to moz-l10n-lint, used in CI
+cl-ext.lang==0.1.0
+compare-locales==7.6.0
```

Comment:
Looking at the pipdeptree, some of these may not need to be here and can be included when they are compiled. But we can tackle that in another pass.

Reply:
Yeah we can thin things out, but I know for sure that `cl-ext.lang` and `compare-locales` should remain first-class/top-level deps else the test and CI steps that use `moz-l10n-lint` will :boom: